app/ios/Podfile.lock

```diff
@@ -954,4 +954,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 557ce83fddb4b3974ada332a9ba0251f0ffcc639
 
-COCOAPODS: 1.13.0
+COCOAPODS: 1.14.3
```

Comment:
I think Clecio wants us to stick with 1.13.0 cause that's what our Gemfile config and CICD uses